partitioned-table.md

```diff
@@ -1302,6 +1302,69 @@ TiDB accesses partitioned tables in one of the two modes: `dynamic` mode and `st
 set @@session.tidb_partition_prune_mode = 'dynamic'
 ```
 
+In `dynamic` mode, TiDB collects table-level summary statistics, or GlobalStats, from partitioned tables. For detailed information about GlobalStats, see [Collect statistics of partitioned tables in dynamic pruning mode](/statistics.md#collect-statistics-of-partitioned-tables-in-dynamic-pruning-mode).
+
+When switching from `static` mode to `dynamic` mode, you need to check and collect statistics manually. This is because after the switch to `dynamic` mode, there are only partition statistics, and no table statistics. GlobalStats are collected only upon the next `auto-analyze` operation.
```

Comment:
```suggestion
When switching from the `static` mode to `dynamic` mode, you need to check and collect statistics manually. This is because after the switch to `dynamic` mode, partitioned tables have only partition-level statistics but no table-level statistics. GlobalStats are collected only upon the next `auto-analyze` operation.
```